browser/package.json

```diff
@@ -1,9 +1,9 @@
 {
   "name": "@extism/runtime-browser",
-  "version": "0.0.1-rc.6",
+  "version": "0.0.1-rc.10",
   "description": "Extism runtime in the browser",
   "scripts": {
-    "build": "node build.js",
+    "build": "node build.js && tsc --emitDeclarationOnly --outDir dist",
```

Comment:
`npm-dts` was not generating valid type defs. this is the best hack i could come up with. let tsc do it instead of relying on esbuild. I could maybe implement this in build.js before release.

Reply:
if it works, it works ;) thank you